README.md

```diff
@@ -1,2 +1,46 @@
-# suitest-js-api-mocha-demo
-Demo application on how to use Suitest JS API with Mocha test runner
+# Suitest JavaScript API demo - Mocha integration
+
+This repository holds a minimal set-up, required to launch [Suitest JS API](https://github.com/SuitestAutomation/suitest-js-api) e2e tests with [Mocha test runner](https://github.com/mochajs/mocha).
+
+## Demo set up for Suitest JS API with Mocha test runner
+
+1. Copy `.suitestrc.dist` file as `.suitestrc` and fill in it's contents 
+with your values according to [documentation](https://suite.st/docs/suitest-api/setup/#environment-setup).
+2. Write your tests in `test/` folder. There is an example in `test/dummy.test.js`.
+3. In terminal run `yarn` or `npm i`.
+4. Run `yarn run interactive` or `npm run interactive` for interactive mode or
+ `yarn run automated` or `npm run automated` for automated.
+
+## Debugging tests with IDE
+
+**Webstorm**
+Create `Node.js` configuration where
+1. _Working directory_ path to project root folder.
+2. _JavaScript file_ `./node_modules/.bin/suitest`.
```

Comment:
@AntonSBilan we dont describe how to debug on windows?

Reply:
This issue just about move demo setup to separate repo, issue about debugging on Windows is in qa required